src/panels/lovelace/editor/hui-dialog-edit-card.ts

```diff
@@ -115,29 +115,33 @@ export class HuiDialogEditCard extends LitElement {
   }
 
   private _handleYamlChanged(ev: YamlChangedEvent): void {
-    this._updatePreview(ev.detail.yaml);
+    this._updatePreview(yaml.safeLoad(ev.detail.yaml));
```

Comment:
This means that we no longer catch YAML parse errors and render a card with invalid YAML. I think that we should have a config value that contains format and value and have the preview element deal with that.